b2sdk/raw_api.py

```diff
@@ -51,6 +51,29 @@
 API_VERSION = 'v2'
 
 
+@unique
+class TokenType(Enum):
+    API = 'api'
+    UPLOAD_PART = 'upload_part'
+    UPLOAD_SMALL = 'upload_small'
+
+
+def set_token_type(token_type):
+    """
+    This is a decorator to identify the type of token to be used in session.
```

Comment:
I would say this the other way, because this is the raw API layer, which should not be aware of the session layer above.

"This is a decorator to identify the type of token that must be passed into a function."